src/platform/src/app/globals.css

```
@@ -92,7 +92,7 @@ body,
 html {
   color: rgb(var(--foreground));
   background: rgb(var(--background));
-  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-family: var(--font-inter), 'Inter', Arial, Helvetica, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)